gulpfile.js

```diff
@@ -43,6 +41,10 @@ var config = {
 };
 
 
+// webpack
+var webpackCompiler = webpack(require('./webpack.config')(config));
```

Comment:
This passes `config` to `webpack.config.js`, so it needs to be below the config.